src/app/containers/collection-page.ts

```diff
@@ -30,10 +31,15 @@ import { Book } from '../models/book';
     }
   `]
 })
-export class CollectionPageComponent {
+export class CollectionPageComponent implements OnInit{
   books$: Observable<Book[]>;
 
-  constructor(store: Store<fromRoot.State>) {
-    this.books$ = store.select(fromRoot.getBookCollection);
+  constructor(private store: Store<fromRoot.State>) {
+    this.books$ = this.store.select(fromRoot.getBookCollection);
```

Comment:
Is it necessary to put `this.` when accessing class properties? Is it because of private? Will it work without `this.`?